doc/nrf/gs_recommended_versions.rst

```diff
@@ -22,6 +22,66 @@ However, there are some Zephyr features that are currently only available on Lin
 
    Before you start setting up the toolchain, install available updates for your operating system.
 
+.. _gs_supported_OS:
+
+Supported Operating Systems
+***************************
+
+The following table shows the operating systems and their various versions that supports the |NCS| tools:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Operating System
+     - x86
```

Comment:
I don't think we support x86 Windows?

Reply:
I added this information from here - https://projecttools.nordicsemi.no/confluence/pages/viewpage.action?spaceKey=WYLND&title=TSP+-+requirements - @kotlarek-nordic - Could you please confirm this.

If this is not the case, I think the confluence page will need an update.

Reply:
@divipillai those, you mentioned, are requirements, which are not yet fully covered by our teams. In my opinion we should discuss that with our TPM before releasing it to the public.